projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.spec.ts

```diff
@@ -287,4 +297,34 @@ describe('AsmMainUiComponent', () => {
       visible: false,
     });
   });
+
+  it("should end session on 'End Session' button click", () => {
+    //agent sign in
+    const agent_login = {
+      userId: 'asagent',
+      password: 'password',
+    };
+    component.loginCustomerSupportAgent(agent_login);
```

Comment:
Is this part required?

Reply:
I don't think so, I'll try to change it.